src/Styleguide/Components/FollowArtistPopover/FollowArtistPopoverRow.tsx

```diff
@@ -64,6 +80,7 @@ class FollowArtistPopoverRow extends React.Component<Props, State> {
         },
         updater: (store: RecordSourceSelectorProxy, data: SelectorData) => {
           const { node } = data.followArtist.artist.related.suggested.edges[0]
+          this.excludedArtistIds.add(node.__id)
```

Comment:
So this is managed locally now. Not exactly clear what is the best way to update parent to make sure that this added artistId is not going to show in other rows....